### PR TITLE
Cypress tests multiple variants 

### LINKED ISSUE #3698: Variants - Cypress test multiple variants
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/pull/3509 we have service variants, but we currently only test 1 per service. 

**Describe the solution you'd like**
Test all variants of services

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #3698 

**Overall change:** 

Make it so variants of services are also covered by the e2es.

**Code changes:**

- added a `name` field to `services.js`in the cypress support files
- used that `name` field to access the correct things in the relevant places

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by PriyaKR — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```
Comment (rhenshaw56) [inline .env]: @oluoluoxenfree why are we exposing this file ?

**Review by rhenshaw56 — COMMENTED** [review]
@oluoluoxenfree the pr doesn't link to any issue, just has the link to this pr

File: .env (hunk 1 above)
Comment (oluoluoxenfree) [reply .env]: Accidentally added as I ran a build to run the e2es 🤦‍♀ Will remove @rhenshaw56

**Comment by amywalkerdev** [comment]
Bunch of test failures on CI?

`errorPage404 - serbianCyr - Canonical Tests for serbianCyr errorPage404 serbianCyr Article Error Page Tests should have correct title & description metadata:
     TypeError: Cannot read property 'cyr' of undefined
      at Context.<anonymous> (http://localhost:7080/__cypress/tests?p=cypress/integration/pages/errorPage404/index.js-573:319:59)`

File: cypress/integration/application/index.js [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
-        cy.testResponseCodeAndType(
-          usesArticlePath
-            ? `/${service}/articles/manifest.json`
-            : `/${service}/manifest.json`,
-          200,
-          'application/json',
-        );
-      });
+    it(`should return a 200 status code for ${service}'s service worker`, () => {
+      console.log(config[service].name);
```
Comment (rhenshaw56) [inline cypress/integration/application/index.js]: you've got a console statement here

File: cypress/integration/pages/testsForAllPages.js [hunk 3]
```diff
@@ -225,43 +226,56 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
...
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.accept,
         );
       });
     });
 
     describe('Header Tests', () => {
-      const hasLocalisedName =
-        appConfig[service][variant].serviceLocalizedName !== undefined;
+      // debugger;
```
Comment (rhenshaw56) [inline cypress/integration/pages/testsForAllPages.js]: commented `debugger`

**Review by rhenshaw56 — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]
👍

**Review by chris-hinds — APPROVED** [review]
👍

**Pull request merged**